support-data-rest/src/test/java/org/uniprot/api/support/data/taxonomy/controller/TaxonomyITUtils.java

```diff
@@ -44,7 +44,7 @@ public static TaxonomyDocument createSolrDoc(long taxId, boolean facet) {
                         .strainsAdd(new TaxonomyStrainBuilder().name("str name").build())
                         .hostsAdd(new TaxonomyBuilder().taxonId(taxId + 2).build())
                         .linksAdd("link value")
-                        .active(true)
+                        .active(facet)
```

Comment:
I agree but this is how we are creating the inactive entry even before my change. Because for search, inactive=false is relevant rest of the fields don't matter.
See code
`@Override
    protected void saveEntry(SaveScenario saveContext) {
        saveEntry(10, true);
        saveEntry(15, true);
        saveEntry(20, false);
    }`
and even test ``defaultSearchCanFindOtherName`` which doesn't return inactive before this change.
If you want i can write a separate method to create inactive entry.